nvme-print.c

```diff
@@ -2925,22 +2941,23 @@ static void json_ctrl_registers(void *bar)
 	pmrmscu = mmio_read32(bar + NVME_REG_PMRMSCU);
 
 	root = json_create_object();
-	json_object_add_value_uint(root, "cap", cap);
+	json_object_add_value_uint64(root, "cap", cap);
```

Comment:
Good catch, but these fixes should be in a separate patch.

Reply:
Moved to separate commit - dea1ddb726dc06fdefd12d57e241e8a17c85caf4